.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+dist: trusty
```

Comment:
I'm not very familiar with the Travis CI build process, but would this limit the builds in any way to running well only on a Ubuntu 14.04 base?